lte/gateway/c/core/oai/tasks/amf/amf_app_state_manager.cpp

```diff
@@ -50,20 +50,27 @@ amf_app_desc_t* get_amf_nas_state(bool read_from_redis) {
  * a thread safe call
  */
 void put_amf_nas_state() {
+  OAILOG_FUNC_IN(LOG_AMF_APP);
   magma5g::AmfNasStateManager::getInstance().write_state_to_db();
+  OAILOG_FUNC_OUT(LOG_AMF_APP);
 }
 
 /**
  * Release the memory allocated for the AMF NAS state, this does not clean the
  * state persisted in data store
  */
-void clear_amf_nas_state() { AmfNasStateManager::getInstance().free_state(); }
+void clear_amf_nas_state() {
+  OAILOG_FUNC_IN(LOG_AMF_APP);
+  AmfNasStateManager::getInstance().free_state();
+  OAILOG_FUNC_OUT(LOG_AMF_APP);
+}
 
 map_uint64_ue_context_t* get_amf_ue_state() {
   return AmfNasStateManager::getInstance().get_ue_state_map();
 }
 
 void delete_amf_ue_state(imsi64_t imsi64) {
+  OAILOG_FUNC_IN(LOG_AMF_APP);
```

Comment:
Remove LOG_MME_APP module in all places for amf task

Reply:
Comment addressed.